app/src/components/MovieCard/index.tsx

```diff
@@ -8,17 +8,18 @@ import * as S from './index.styles';
 
 type MovieCardProps = {
     movie: Movie;
+    onClick: () => void;
 };
 
-const MovieCard = ({ movie }: MovieCardProps): JSX.Element => {
+const MovieCard = ({ movie, onClick }: MovieCardProps): JSX.Element => {
```

Comment:
Da li se ovo onClick koristi negde? Ako da, sta se desi ako korisnik klikne na imdb link u kartici?

Reply:
prvobitno mi je admin dash koristio ove komponente pa sam dodao on-click, sada vise ne. Svakako mislim da si ti napravio na svojoj grani handler koji vodi na movie details, tako da odavde cu da smaknem

Reply:
Imao sam jednom taj problem kada su dve klikabilne povrsine jedna na drugoj. U principu moze da se resi tako sto dodas useRef nad ugnjezdenom komponentom i onda proveravas da li e.target.value.contains(tajRef).
Medjutim, ja bih ostavio to tvoje, da bude samo na nazivu filma iz dva razloga:
1. Lakse je
2. Onaj hover efekat koji sam stavio na karticu je apsolutno odvratan i ne bi bilo lose smaknuti ga

Reply:
U redu, sredicemo to sledeci sprint kad sve ostalo bude radilo